docs/Developers.md

```diff
@@ -6,6 +6,7 @@
   * [Building SRT](#building-srt)
 * [Project Structure](#project-structure)
 * [Coding Rules](#rules)
+* [Generated files](#generated-files)
```

Comment:
I would rather move it to the very end of the document as it is a very rare case that someone (especially external) would use those scripts.

Reply:
Right, of course.